elements/lisk-transactions/src/utils/verify.ts

```diff
@@ -54,14 +59,14 @@ export const verifyMinRemainingBalance = (
 };
 
 export const verifyAccountNonce = (
-	id: string,
+	id: Buffer,
 	account: Account,
 	nonce: bigint,
 ): TransactionError | undefined => {
 	if (nonce < account.nonce) {
 		return new TransactionError(
 			`Incompatible transaction nonce for account: ${
-				account.address
+			bufferToHex(account.address)
```

Comment:
isnt it just account.address.toString('hex')

Reply:
could do that too.. just used to utility, should i change?

Reply:
not necessary to change =)